xdsl/xdsl_opt_main.py

```diff
@@ -329,7 +329,11 @@ def register_all_targets(self):
 
         def _output_mlir(prog: ModuleOp, output: IO[str]):
             printer = Printer(
-                stream=output, print_generic_format=self.args.print_op_generic
+                stream=output,
+                print_generic_format=self.args.print_op_generic,
+                print_debuginfo=self.args.print_debuginfo
+                if "print_debuginfo" in self.args
+                else False,
```

Comment:
Can you add an option for `print_debuginfo` instead? So it appears in `--help` for instance?

Reply:
I looked for this but have no idea where this is happening. Any pointer?